draft-ietf-rats-architecture.md

```diff
@@ -134,6 +134,35 @@ Appraisal Policy to make application-specific decisions such as authorization de
 The Attestation Result Appraisal Policy might, for example, be configured in the Relying Party 
 by an administrator.
 
+## Composite Remote Attestation Data Flow
+
+Many devices are composed of one or more leader units and one or more subordinate units. The leader units are the control or management units and have management interfaces to communicate with outer systems like the Network Management System (NMS). The other subordinate units are for some specific functions and have no management interfaces to communicate outside but can only be managed by the leader units by some kind of inner links. The leader units and subordinate units all have some kind of trusted computing base to measure their trustworthiness. These devices can be called composite devices, and the trustworthiness of the device is therefore only accurately represented by a composite measure that includes all its leader units and subordinate units. A typical example is the carrier-grade switches and routers that have the main control boards as the leader units and the line-processing boards and feature-processing boards as the subordinate units.
```

Comment:
Add line breaks to keep source lines short

Reply:
done, thanks